agenda/domain/src/main/java/com/nibalk/tasky/agenda/domain/usecase/FormatProfileNameUseCase.kt

```diff
@@ -0,0 +1,13 @@
+package com.nibalk.tasky.agenda.domain.usecase
+
+class FormatProfileNameUseCase() {
+    operator fun invoke(fullName: String): String {
+        val names = fullName.trim().split(" ")
+        return when (names.size) {
+            0 -> ""
+            1 -> names[0].take(2).uppercase()
+            2 -> "${names[0][0]}${names[1][0]}".uppercase()
+            else -> "${names[0][0]}${names[names.size - 1][0]}".uppercase()
```

Comment:
I think these two cases are equivalent, aren't they? In both cases, you want to pick the first letter of the first word and the first letter of the last word.